packages/connection/tests/php/test-rest-endpoints.php

```diff
@@ -159,6 +212,12 @@ function( $plugin ) {
 	 * Testing the `connection/reconnect` endpoint, full reconnect.
 	 */
 	public function test_connection_reconnect_full() {
+		$deregister_request_filter = function() {
+			return false;
+		};
+
+		add_filter( 'jetpack_connection_disconnect_site_wpcom', $deregister_request_filter );
```

Comment:
If you'd like to shorten this test, I think you can use [`__return_false`](https://developer.wordpress.org/reference/functions/__return_false/) as the callback.